.pre-commit-config.yaml

```diff
@@ -19,7 +19,8 @@ repos:
         stages: [pre-commit]
       - id: end-of-file-fixer
         stages: [pre-commit]
-        exclude: \.((golden\.adoc)|svg)$
+        # exclude read-only files, and those sensitive to any change
```

Comment:
I'm not sure this is the best description of the files described.
"Exclude reference and generated files" ?